scipy/sparse/_lil.py

```diff
@@ -112,7 +112,11 @@ def count_nonzero(self):
     count_nonzero.__doc__ = _spbase.count_nonzero.__doc__
 
     def __str__(self):
-        val = ''
+        val = repr(self)
+        if self.nnz == 0:
+            return val
+
+        val += '\n  Coords\tValue\n'
```

Comment:
The base class uses Coords and Values, so we should use Values here too.

Though I wonder: since this doesn't support maxprint, should we even have this override?

Reply:
Good point -- This `str` override could print something **very** long. And the base class will handle it just fine. 

I've removed that method from `_lilbase` here.

I've also rebased to include the csr-1d stuff in case that changes anything.